Content.Shared/SS220/OrigamiBook/SharedOrigamiSystem.cs

```diff
@@ -38,8 +42,15 @@ private void OnThrowHit(Entity<OrigamiWeaponComponent> ent, ref ThrowDoHitEvent
         if (!HasComp<OrigamiUserComponent>(args.Component.Thrower))
             return;
 
-        if (TryComp<IdentityBlockerComponent>(args.Target, out var identityBlocker)
-            && identityBlocker.Coverage is IdentityBlockerCoverage.EYES or IdentityBlockerCoverage.FULL)
+        if (_gun.TryGetGun(args.Component.Thrower.Value, out _, out _))
+            return;
+
+        var hasEyeProtection =
+            _inventory.TryGetSlotEntity(args.Target, "eyes", out var eyes) && IsEyeBlocker(eyes.Value) ||
+            _inventory.TryGetSlotEntity(args.Target, "mask", out var mask) && IsEyeBlocker(mask.Value) ||
+            _inventory.TryGetSlotEntity(args.Target, "head", out var head) && IsEyeBlocker(head.Value);
```

Comment:
Слоты для проверки вынести в поле компонента/константу либо проверять все имеющиеся слоты у энтити
```suggestion
        var hasEyeProtection = false;
        foreach (var slotName in ent.Comp.BlockerSlots)
        {
            if (_inventory.TryGetSlotEntity(args.Target, slotName, out var slot) && IsEyeBlocker(slot.Value))
            {
                hasEyeProtection = true;
                break;
            }
        }
```